plugins/azure/sqlserver/sqlServerTlsVersion.js

```diff
@@ -70,5 +76,49 @@ module.exports = {
         }, function() {
             callback(null, results, source);
         });
+    },
+    remediate: function(config, cache, settings, resource, callback) {
+        var remediation_file = settings.remediation_file;
+        var putCall = this.actions.remediate;
+
+        // inputs specific to the plugin
+        var pluginName = 'sqlServerTlsVersion';
+        var baseUrl = 'https://management.azure.com/{resource}?api-version=2020-08-01-preview';
+        var method = 'PATCH';
+
+        // for logging purposes
+        var serverNameArr = resource.split('/');
+        var serverName = serverNameArr[serverNameArr.length - 1];
+
+        // create the params necessary for the remediation
+        if (settings.region) {
+            var body = {
+                'location': settings.region,
+                'properties': {
+                    'minimalTlsVersion': '1.2'
+                }
+            };
+
+            // logging
+            remediation_file['pre_remediate']['actions'][pluginName][resource] = {
+                'TLS1.2': 'Disabled',
+                'Server': serverName
+            };
+
+            helpers.remediatePlugin(config, method, body, baseUrl, resource, remediation_file, putCall, pluginName, function(err, action) {
+                if (err) return callback(err);
+                if (action) action.action = putCall;
+
+
+                remediation_file['post_remediate']['actions'][pluginName][resource] = action;
+                remediation_file['remediate']['actions'][pluginName][resource] = {
+                    'Action': 'Enabled'
```

Comment:
Here dont we need to mention the details like TLS1.2 :Enabled ,Server: servername